tests/quality_pragmaitc/test_refinement.py

```diff
@@ -306,40 +297,39 @@ def test_aGoalOrDecomposedWithTwoTasksMayNotBeAchievable():
     interp.addQualityConstraint(qc)
 
     plan = goal.isAchievable(fullContext, interp)
-    assert goal.decomposition
+    assert goal.decomposition is Decomposition.OR
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4895731)